tests/cli/bootprep/input/test_session_template.py

```diff
@@ -92,10 +92,12 @@ def get_input_and_expected_bos_data(self, name='my-session-template',
         bos_payload_bootsets = {}
         for boot_set_name, boot_set_arch in arch_by_bootset.items():
             input_bootsets[boot_set_name] = {
-                'node_roles_groups': ['Compute']
+                'node_roles_groups': ['Compute'],
+                'rootfs_provider_passthrough': "dvs:api-gw-service-nmn.local:300:hsn0,nmn0:0"
             }
             bos_payload_bootsets[boot_set_name] = {
                 'node_roles_groups': ['Compute'],
+                'rootfs_provider_passthrough': "dvs:api-gw-service-nmn.local:300:hsn0,nmn0:0",
```

Comment:
It would be good to add a test where the rootfs_provider_passthrough has a variable that needs to be rendered with Jinja2